README.md

```diff
@@ -49,18 +50,40 @@ Note by setting `tos_accepted` to *true* implies that you have read and accepted
 and terminates challenge only for certain domains, please refer to the
 [Plugin Config](#plugin-config) section.
 
-#### Trigger creation of certificate
+#### Create certificates
 
 Assume Kong proxy is accessible via http://mydomain.com and https://mydomain.com.
 
 ```bash
+# Trigger asynchronous creation from proxy requests
+# The following request returns immediately with Kong's default certificate
+# Wait up to 1 minute for the background process to finish
 $ curl https://mydomain.com -k
-# Returns Kong's default certificate
-# Wait up to 1 minute
+
+# OR create from Admin API with version >= 0.2.4
+# User can also use this endpoint to force "renew" a certificate
+$ curl http://localhost:8001/acme/create -d host=mydomain.com
+
+# Furthermore, it's possible to run sanity test before creating any certificate
+$ curl http://localhost:8001/acme/create -d host=mydomain.com -d test_only=1
```

Comment:
it basically sends a request to `http://mydomain.com/.well-known/acme-challenge/x` to verify the kong setup. it's not perfect but basically automate the "troubleshooting" i've been through when answering issues.

Reply:
it will also interprete the response to catch some common setup error: https://github.com/Kong/kong-plugin-acme/blob/master/kong/plugins/acme/api.lua#L42

Reply:
Please document this behavior in the readme.